### PR TITLE
[TD-1998]: Restore all files tracked by sync automation.

### PR DESCRIPTION
**Pull request by asutosh**
## Description

Restores ci/*, .github/workflows/{release.yml,api-tests.yml,ci-tests.yml} to the state before the
commit 0e27df86ae5f9808158b4a875fd03b357b7bef48 (go-1.19 changes automatically picked by sync-automation from master)

## Related Issue
TT-9199

### CODE REVIEW

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/5177/merge
Commit:  
Triggered by: pull_request (@asutosh)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/5323224868)

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/5177/merge
Commit: bf467b4bf18b26723ce8daafd78345ca70068366 
Triggered by: pull_request (@asutosh)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/5453712951)

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/5177/merge
Commit: ada86d01c9251eb3cb6644e8711732a304808343 
Triggered by: pull_request (@asutosh)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/5453716032)

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/5177/merge
Commit: 44ff375b763db3ce4b807c577b642ce197cb84b5 
Triggered by: pull_request (@asutosh)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/5453874729)

File: .github/workflows/api-tests.yml [hunk 1]
```diff
@@ -67,15 +69,15 @@ jobs:
...
-          echo "branch=${{ github.head_ref	}}" >> $GITHUB_OUTPUT
+          echo "::set-output name=branch::${{ github.head_ref	}}"
         fi
         if [ ! -z "${{ github.base_ref	}}" ] && git ls-remote --exit-code --heads https://${TOKEN}@github.com/TykTechnologies/tyk-analytics ${{ github.base_ref	}}; then
-          echo "branch=${{ github.base_ref	}}" >> $GITHUB_OUTPUT
+          echo "::set-output name=branch::${{ github.base_ref	}}"
         fi
         if [ ! -z "${{ github.ref	}}" ] && git ls-remote --exit-code --heads https://${TOKEN}@github.com/TykTechnologies/tyk-analytics ${{ github.ref }}; then
-          echo "branch=${{ github.ref	}}" >> $GITHUB_OUTPUT
+          echo "::set-output name=branch::${{ github.ref	}}"
```
Comment (ermirizio) [inline .github/workflows/api-tests.yml:80]: Revert this to $GITHUB_OUTPUT please

**Review by ermirizio — CHANGES_REQUESTED** [review]

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/5177/merge
Commit: bcd90370b33829d8fbb492f4ff1f5c5d74c0047d 
Triggered by: pull_request (@asutosh)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/5455296862)

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/5177/merge
Commit: 6b9053be4c3ab7842924791fa4b476afc357cde2 
Triggered by: pull_request (@asutosh)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/5462727329)

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/5177/merge
Commit:  
Triggered by: pull_request (@asutosh)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/5462758604)

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/5177/merge
Commit: 55d28b46e9f52906f8612fdb3f78c0af52657ceb 
Triggered by: pull_request (@asutosh)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/5462953322)

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/5177/merge
Commit: a9007d4a9e89ceab4d1a0b8bb385efa3ee8deb67 
Triggered by: pull_request (@asutosh)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/5463122892)

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/5177/merge
Commit: 7ef0d376e7eb29d21cd17b52538435260cf41848 
Triggered by: pull_request (@ermirizio)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/5476883559)

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/5177/merge
Commit: f9f3a42ecb7d29d07b293e90a3da2c9677b41716 
Triggered by: pull_request (@asutosh)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/5530172402)

**Review by titpetric — COMMENTED** [review]
Some locations (github, ci/goreleaser, ci/images/plugin-compiler) have healthy versions on 4.0.14

**Pull request closed without merging**